nchelpers/__init__.py

```diff
@@ -918,7 +919,7 @@ def var_bounds_and_values(self, var_name, bounds_var_name=None):
             )
             return zip(midpoints[:-1], values, midpoints[1:])
 
-    def var_range(self, var_name):
+    def var_range(self, var_name, chunksize=1000000):
```

Comment:
Want to say 2**20 here? Easier to read as 1MB than having to count out the number of zeros.